hack/common.sh

```diff
@@ -1001,6 +1001,23 @@ function os::build::find-binary() {
 }
 readonly -f os::build::find-binary
 
+# os::build::get-go-bin-location-from-gopath locates a go install binary in GOPATH directories 
+# and returns the first path where the binary was found. 
+function os::build::get-go-bin-location-from-gopath() {
+  local matching_gopath=""
+
+  gopaths=`echo $GOPATH | sed 's/:/ /g'`
+  for gopath in $gopaths
```

Comment:
I think changing the internal field separator would be a cleaner way of iterating over the parts:
```sh
IFS=':'
for part in $GOPATH; do
    # logic
done
```